### PR TITLE
Add engagement duration stats endpoint

### PR DESCRIPTION
**Pull request by indrekj**
https://github.com/salemove/web_services/pull/595

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1364,6 +1364,44 @@ Possible dimensions that can be used in filters:
     ]
 
 
+### Engagement Duration [POST /engagements/stats/duration]
+Returns sum of engagement durations in seconds.
```
Comment (deiwin) [inline apiary.apib]: the sum

**Comment by deiwin** [comment]
+1

File: apiary.apib [hunk 2]
```diff
@@ -1364,6 +1364,44 @@ Possible dimensions that can be used in filters:
...
+
+Possible dimensions that can be used in filters:
+* `site_id` - Site ID
+* `operator_id` - Operator ID
+
++ Request (application/json)
+
+  + Headers
+
+      Authorization: Token 2pFuPlTYkH7T4MylrwxApA
```
Comment (stenver) [inline apiary.apib]: So we went with `Token` not with `ApiToken`?

Comment (indrekj) [inline apiary.apib]: Yes, both are supported but `Token` is preferred

**Comment by stenver** [comment]
!squash

**Pull request merged**